GoogleChromeLabs/ps-analysis-tool
# Add support forum link to PS landing page and dashboard

## Description

Add support forum link to Privacy Sandbox landing page and dashboard pages for easy access.

## Relevant Technical Choices

- Create a new component for the link and conditionally add to `LandingPage` component.

## Testing Instructions

- Go to Privacy Sandbox landing page and dashboard pages and see if they have links available and they go to the forum in a new tab.

## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #846
